sycl/source/detail/program_manager/program_manager.hpp

```diff
@@ -74,9 +75,11 @@ class ProgramManager {
   ///        current specialization constants settings; can be nullptr.
   ///        Passing as a raw pointer is OK, since it is not captured anywhere
   ///        once the function returns.
+  /// \param FromProgramApi indicates that it came from a public program API
```

Comment:
Does it sound better this way?
```suggestion
  /// \param FromProgramApi indicates that call comes from a public program API.
```

Reply:
Yeah, thank you!